Minor updates to rfc

JanLin opened the pull request:
Updating overview diagram and minor updates.

kdenhartog commented:
@JanLin It looks like DCO is preventing this work from being merged. Checkout this [guide](https://github.com/hyperledger/indy-sdk/blob/5f9144feab5b9e969389e325da47d7218f87081e/docs/contributors/signing-commits.md) for details of how to add a DCO that passes the check.

JanLin commented:
> @JanLin It looks like DCO is preventing this work from being merged. Checkout this [guide](https://github.com/hyperledger/indy-sdk/blob/5f9144feab5b9e969389e325da47d7218f87081e/docs/contributors/signing-commits.md) for details of how to add a DCO that passes the check.

I believe it is now fixed. Let me know if addresses your comment.
Thanx

dhh1128 commented:
I re-built this PR from scratch, cherry-picking @JanLin 's commits beginning with 3016c4d, in PR #202 . That merged cleanly. I'm now closing this PR as redundant. Jan: before submitting any further PRs, I suggest you reset your fork and pull from upstream/master so your history eliminates commit 8bc32f7 and contains commit 89c4e56. This, plus religious use of the `-s` switch in conjunction with the `git commit` command, will streamline future PRs.
